sphinx/api/configuration/metadata.rst

```diff
@@ -685,6 +685,40 @@ The metadata options for a configuration fall into four categories:
          expression syntax can be used with non-numeric variables in the same
          way as the fail-if metadata.
 
+         .. _trigger-file-creation:
+
+         It is possible to use metadata triggers to trigger file creation, 
+         switching on/off as required.
+
+         For example, with the following rose-app.conf and rose-meta.conf 
```

Comment:
Not a blocker, but for clarity & consistency it is nice to keep literal identifiers such as file names in monospace font, i.e:

```suggestion
         For example, with the following ``rose-app.conf`` and ``rose-meta.conf``
```

Reply:
Or better still:

```rst
:rose:file:`rose-app.conf`
```

Reply:
Thanks! I'll change this now :)